src/adapter/src/catalog.rs

```diff
@@ -2719,6 +2720,22 @@ impl Catalog {
                 }
             }
         }
+        // Operators aren't stored in the catalog, but we would like them in
+        // introspection views.
```

Comment:
Does this comment mean that nothing in `OP_IMPLS` is stored in the in memory catalog?

Reply:
Yes. Operators don't (yet) have GlobalIds, and are (were) only ever looked it in the `resolve_op` function. They thus aren't a `CatalogItem`, hence them needing to be special. They probably should be, but I'm doing a smaller set of work here.

Reply:
Ah ok. Let's open an issue if there isn't one already, but this PR seems strictly better than before so I think it's fine.

Reply:
https://github.com/MaterializeInc/materialize/issues/17835